src/utils/data_loader.rs

```diff
@@ -48,11 +43,11 @@ fn map_restaurant_json2insert(input: &JsonRestaurant) -> InsertRestaurant{
     }
 }
 
-fn map_smileyreport_json2insert(input: &JsonSmileyReport, res_id: i32) -> InsertSmileyReport{
+fn map_smileyreport_json2insert(input: &JsonSmileyReport, res_id: i32) -> InsertSmileyReport {
     InsertSmileyReport {
         date: (*input.date).to_string(),
         smiley: input.smiley,
         restaurant_id: res_id,
         report_id: (*input.report_id).to_string(),
     }
-}
\ No newline at end of file
+}
```

Comment:
Refrain from adding a newline here. I disturbs the PR.

Reply:
blamer codes formatting setting